docs/Model_Assumptions.rst

```diff
@@ -52,19 +57,50 @@ The parameters `dispatch_price`, `efficiency` and `installedCap` of transformers
 This means that these parameters need to be given for the electrical output power in case of a diesel generator (more examples: electrolyzer - H2, heat pumps and boiler - nominal heat ouput, inverters / rectifiers - electrical output power).
 This also means that the costs of the fuel of a diesel generator (input flow) are not included in its `dispatch_price` but in the `dispatch_price` of the fuel source.
 
-Energy excess
-#############
 
-An energy excess sink is placed on each of the LES energy busses, and therefore energy excess is allowed to take place on each bus of the LES.
-This means that there are assumed to be sufficient vents (heat) or transistors (electricity) to dump excess (waste) generation.
-Excess generation can only take place when a non-dispatchable source is present or if an asset can supply energy without any fuel or dispatch costs.
+Energy providers (DSOs)
```

Comment:
```suggestion
Energy providers
```
I have come to know that DSO is not the same as energy provider, so we can drop that specification. The text below, if it also mentions DSO, should also be changed.